hunts/models.py

```diff
@@ -22,6 +26,14 @@ def save(self, *args, **kwargs):
             self.slug = slugify(self.name)
         super().save(*args, **kwargs)
 
+        # adds default start and end times if not provided
+        if not self.start_time:
+            self.start_time = self.created_on
+            super().save(*args, **kwargs)
+        if not self.end_time:
+            self.end_time = self.start_time + timedelta(days=3)
```

Comment:
Actually this is probably best implemented as a `post_save()` method: https://docs.djangoproject.com/en/3.1/ref/signals/#post-save

It even has a `created` param which you can use so it only triggers on new Hunt creations.